Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -564,22 +564,8 @@ - (void)uploadAvatarToService:(NSData *)avatarData
                             @"%@ avatar upload progress: %.2f%%", self.tag, uploadProgress.fractionCompleted * 100);
                     }
                     success:^(NSURLSessionDataTask *_Nonnull uploadTask, id _Nullable responseObject) {
-                        OWSAssert([uploadTask.response isKindOfClass:[NSHTTPURLResponse class]]);
-                        NSHTTPURLResponse *response = (NSHTTPURLResponse *)uploadTask.response;
-
-                        // We could also construct this URL locally from manager.baseUrl + formKey
-                        // but the approach of getting it from the remote provider seems a more
-                        // robust way to ensure we've actually created the resource where we
-                        // think we have.
-                        NSString *avatarUrlPath = response.allHeaderFields[@"Location"];
-                        if (avatarUrlPath.length == 0) {
-                            OWSProdFail(@"profile_manager_error_avatar_upload_no_location_in_response");
```

Comment:
I know you eliminated this prod fail, but please use `SignalServiceKit/Utilities/extract_analytics_event_names.py` to convert analytics keys into code-generated analytics event name methods.

Reply:
Done!